packages/koa-metrics/README.md

```diff
@@ -0,0 +1,74 @@
+# `@shopify/koa-metrics`
+
+Performance metrics implemented with DogStatsD (<https://docs.datadoghq.com/developers/dogstatsd/>) are used widely across services at Shopify. The standard metrics should be documented and following strong conventions.
```

Comment:
I don't think we want to talk about Shopify here or 'should's, we should instead say what the package offers, so maybe something like 

'Opinionated performance metric tracking for Koa, implemented with DogStatsD'